spharpy/samplings/coordinates.py

```diff
@@ -303,3 +354,36 @@ def n_max(self, value):
             self._n_max = None
         else:
             self._n_max = int(value)
+
+    @property
+    def radius_atol(self):
```

Comment:
I find the name of the property a bit confusing. How about renaming it to simply `radius_tolerance` ?
There is not rtol, which would make the distinction necessary and the docstring also does not really help to clarify what atol means.

Reply:
agreed and changed